src/transaction.rs

```diff
@@ -11,18 +11,15 @@ use std::mem::ManuallyDrop;
 use super::{connection::Connection, statement::Statement};
 use crate::{connection::stmt_cache::StmtCacheData, statement::StatementData, Execute, Queryable};
 
-pub struct Transaction<'c, C>
+pub struct Transaction<'c, C: FirebirdClient>
 where
     C: FirebirdClient,
```

Comment:
```suggestion
pub struct Transaction<'c, C>
where
    C: FirebirdClient,
```

The trait bound is repeated